security/vertx-jwt/src/main/java/io/quarkus/ts/security/vertx/config/VertxAuthObjectMapperCustomizer.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.ts.security.vertx.config;
+
+import javax.inject.Singleton;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+
+import io.quarkus.jackson.ObjectMapperCustomizer;
+
+@Singleton
+public class VertxAuthObjectMapperCustomizer implements ObjectMapperCustomizer {
+
+    @Override
+    public void customize(ObjectMapper objectMapper) {
+        // TODO https://github.com/quarkusio/quarkus/issues/16555
```

Comment:
The issue has been fixed, so this TODO should be addressed.

Reply:
The way that was resolved is fine but we have created a new issue in order to improve the usability